docs/integrations/azure/locations.md

```diff
@@ -52,13 +76,18 @@ integrations:
 
 The configuration is a structure with these elements:
 
-- `host`: The DevOps host; only `dev.azure.com` is supported.
-- `token` (optional): A personal access token as expected by Azure DevOps.
-- `credential`: (optional): A service principal or managed identity
+- `credentials`: (optional): A service principal or managed identity
+
+The `credentials` element is a structure with these elements:
+
+- `organizations`: (optional): A list of organizations for which this credential should be used. Only supported for Azure DevOps (cloud). If not specified the credential will be used for all organizations for which no other credential is configured.
```

Comment:
Why would this be cloud only? This should work with Server as well but only with a PAT or am I missing something here? Would be really disappointing if that is the case

Reply:
If this does work with Server we should mention that Organizations is synonymous with Team Project Collections

Reply:
I cannot confirm this because I do not have access to an Azure DevOps Server. But from what I remember, a personal access token is server-wide on-premise, depending on the user's permissions. You can already add multiple Azure DevOps servers because they have different host names, so I don't see a need for separate credentials per collection.

Reply:
I did not implement support for the team project collection URLs in the `DefaultAzureDevOpsCredentialsProvider` but that would be simple to add if it does make sense.